src/i18n/index.ts

```diff
@@ -1 +1,21 @@
-export * from './fr'
+import { useSelector } from 'react-redux'
+import { selectLocale } from '../core/useCases/locale'
+import { Locale } from '../core/useCases/locale/locale.reducer'
```

Comment:
Alors pour les useCase, on ne doit pas importer plus loin que `useCases/<USE_CASE>`.
Il faut ensuite exporter dans l'index du useCase ce qu'on autorise à être utilisé de l'exétieur.

Autre choses, les imports relatif ne sont que pour les fichiers qui ont un lien direct entre eux, au sein d'un même use case ou d'un même container. Pour le reste, on fait des imports à partir de `src`

```suggestion
import { Locale } from 'src/core/useCases/locale/locale'
```